main.cpp

```diff
@@ -188,68 +191,89 @@ struct Guitar
 
 };
 
-/*
-Thing 2) wind mill
-5 properties:
-    1) number of blades (unsigned int)
-    2) rotational speed (float)
-    3) pole height (float)
-    4) amount of energy generated in a day (float)
-    5) blade length (unsigned int)
-3 things it can do:
-    1) convert mechanical energy into electrical energy
-    2) rotate their blades
-    3) connect to the electrical grid
- */
+bool Guitar::String::isCoated()
+{
+    return coated;
+}
+
+void Guitar::String::setTension(float localTension)
+{
+    tension = localTension;
+}
+
+float Guitar::String::getGauge()
+{
+    return gauge;
+}
+
+void Guitar::playChord(std::string chord)
+{
+    chordToPlay = chord;
+}
+void Guitar::PlaySolo()
+{
+    // do nothing
+}
+bool Guitar::connectToAmplifier()
+{
+    bool isConnectedToAmp = true;
+    return isConnectedToAmp;
```

Comment:
write something else here.  What's the point of having a function that only returns true?
functions perform work.
perform some work/logic in this function.  add function parameters if needed
- [ ] revise